sbin/common-functions.sh

```diff
@@ -28,7 +28,7 @@ determineBuildProperties() {
 
     BUILD_TYPE=normal
     DEFAULT_BUILD_FULL_NAME=${OS_KERNEL_NAME}-${OS_MACHINE_NAME}-${BUILD_TYPE}-${JVM_VARIANT}-release
-    BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
+    export BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
```

Comment:
Just checked `makejdk-any-platform.sh` is the entry point, but if its not set there, it never gets set, so `export` keyword is important for setting a default value